dashboard/public/js/main.js

```diff
@@ -103,3 +103,69 @@ function firstClientVisit() {
 		return true;
 	}
 }
+
+var textBox = document.getElementById("password-box");
+var password = document.getElementById('password');
```

Comment:
Don't define global variables here. Refer to [this](https://github.com/NikhilM98/sugarizer-school-portal-server/pull/32/files#r644055897) comment